Tests/Foundation/TestBundle.swift

```diff
@@ -42,8 +42,16 @@ internal func testBundleName() -> String {
     return testBundle().infoDictionary!["CFBundleName"] as! String
 }
 
-internal func xdgTestHelperURL() -> URL {
+internal func xdgTestHelperURL() throws -> URL {
+    #if os(Windows)
+    // Adding the xdgTestHelper as a dependency of TestFoundation causes its object files (including the main function) to be linked into the test runner executable as well
+    // While this works on Linux due to special linker functionality, this doesn't work on Windows and results in a collision between the two main symbols
+    // SwiftPM also cannot support depending on this executable (to ensure it is built) without also linking its objects into the test runner
+    // For those reasons, using the xdgTestHelper on Windows is currently unsupported and tests that rely on it must be skipped
+    throw XCTSkip("xdgTestHelper is not supported during testing on Windows")
```

Comment:
Hmm, this was usable when we were running the tests through CMake. I think that the issue is limited to SPM which doesn't build the libraries and tries to link the executable object library into another executable. Can we clarify this to be unsupported when running with SPM?

Reply:
Yes this is a limitation of SwiftPM that I've brought up to some of the SwiftPM folks - I see we have a few GitHub issues tracking it. SwiftPM attempts to link the object files from the executable into the test runner, and while this happens to work on Linux on Windows the two `main` functions have conflicting symbols and linking fails.

> Can we clarify this to be unsupported when running with SPM?

Is there something that you'd like me to add in addition to the comment that I added here? I think that comment should explain why SwiftPM cannot support this but I'm happy to add any details that you think are missing.

Reply:
Can we update the message of the `XCTSkip` to call out that this is unsupported with SPM?

Reply:
Sure, added in 595a4e9